app/Models/V2/ProjectPipeline.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Models\V2;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class ProjectPipeline extends Model
+{
+    use HasFactory;
+    use SoftDeletes;
+
+    public $table = 'project_pipeline';
+
+    protected $fillable = [
+        'Name',
+        'SubmittedBy',
+        'Description',
+        'Program',
+        'Cohort',
+        'PublishFor',
+        'URL',
+        'CreatedDate',
+        'ModifiedDate',
```

Comment:
Please move these columns to snake_case, as is the norm for all tables in our DB.

Reply:
I'm also surprised that I don't see a migration to create the table in this PR. Has it already been merged? Typically the creation of the model .php and the migration to create the DB table would be part of the same PR.

Reply:
Also, it's odd to me that this would have custom `CreatedDate` and `ModifiedDate` columns when almost all of our models use the built-in Laravel `created_at` and `updated_at`, which get set automatically.